### PR TITLE
[MKC-1703] Fixes for the Micropython course

### PR DESCRIPTION
**Pull request by BenjaminDannegard**
## What This PR Changes
- Fixed a broken script
- Fixed text that was wrong

## Contribution Guidelines
- [ ] I confirm that I have read the [contribution guidelines](https://github.com/arduino/docs-content/tree/main/contribution-templates) and comply with them.

### CODE REVIEW

File: content/micropython/02.micropython-course/course/07.internet-of-things/01.internet-of-things.md
Comment (jacobhylen) [inline content/micropython/02.micropython-course/course/07.internet-of-things/01.internet-of-things.md:1]: There are still some problems in this file. 
- There are 2 "exercise 2"s
- I can't get the "make a request" section to work on my Nano ESP32
- In Exercise 1 there is a sentence on line 98 that doesn't make any sense. 
- The code block that starts on line 100 tries to use the variables "WIFI_NETWORK" and "WIFI_PASSWORD" but they are not defined here.

File: content/micropython/02.micropython-course/course/03.python-cc/python-cc.md
Comment (jacobhylen) [inline content/micropython/02.micropython-course/course/03.python-cc/python-cc.md:1]: Looks good.

**Review by jacobhylen — CHANGES_REQUESTED** [review]
Looks good, but while reviewing I found that there more many issues in one of the tutorials, and while we're at it let's fix it properly.

File: content/micropython/02.micropython-course/course/07.internet-of-things/01.internet-of-things.md
Comment (BenjaminDannegard) [reply content/micropython/02.micropython-course/course/07.internet-of-things/01.internet-of-things.md:1]: Need to test and fix the "make a request" section, everything else is fixed

**Review by jacobhylen — APPROVED** [review]
I realised that the reason I couldn't get the make a request section to work was user error :-) 
All is good. LGTM.

**Pull request merged**